pallene/translator.lua

```diff
@@ -31,77 +31,124 @@ local Translator = util.Class()
 function Translator:init()
     self.last_index = 1
     self.partials = {}
+    self.exports = {}
+    self.input = false
```

Comment:
Please put the declaration of `self.input` next to the declaration of `last_index`. It would also be a good idea to add comments next to each of these instance variables, describing their types. (The coder.lua should have some examples of that).